cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/NamespaceHttpHandlerTest.java

```diff
@@ -167,6 +167,23 @@ public void testCreateDuplicate() throws Exception {
   }
 
   @Test
+  public void testCreateWithConfig() throws Exception {
```

Comment:
Should we also add a check that verifies that the name from the json meta is not used in the update properties API?